nca/NetworkConfig/NetworkLayer.py

```diff
@@ -212,18 +286,54 @@ def _allowed_xgress_conns(self, from_peer, to_peer, is_ingress):
         return PolicyConnections(captured_res, allowed_conns, denied_conns,
                                  all_allowed_conns=allowed_conns | allowed_non_captured_conns)
 
+    def _allowed_xgress_conns_optimized(self, is_ingress, peer_container):
+        res_conns = self.collect_policies_conns_optimized(is_ingress)
+        # Note: The below computation of non-captured conns cannot be done during the parse stage,
+        # since before computing non-captured conns we should collect all policies conns
+        # compute non-captured connections
```

Comment:
```suggestion
        # since before computing non-captured conns we should collect all policies conns

        # compute non-captured connections
```